tests/validation/tests/v3_api/test_auth.py

```diff
@@ -36,6 +40,22 @@
     os.environ.get("RANCHER_OPENLDAP_SPECIAL_CHAR_PASSWORD")
 FREEIPA_SPECIAL_CHAR_PASSWORD = \
     os.environ.get("RANCHER_FREEIPA_SPECIAL_CHAR_PASSWORD")
+OPENLDAP_HOSTNAME_OR_IP_ADDRESS = \
+    os.environ.get("RANCHER_OPENLDAP_HOSTNAME_OR_IP_ADDRESS")
+OPENLDAP_SERVICE_ACCOUNT_NAME = \
+    os.environ.get("RANCHER_OPENLDAP_SERVICE_ACCOUNT_NAME")
+OPENLDAP_SERVICE_ACCOUNT_PASSWORD = \
+    os.environ.get("RANCHER_OPENLDAP_SERVICE_ACCOUNT_PASSWORD")
+OPENLDAP_USER_SEARCH_BASE = os.environ.get("RANCHER_OPENLDAP_USER_SEARCH_BASE")
+OPENLDAP_AUTH_USER_PASSWORD = \
+    os.environ.get("RANCHER_OPENLDAP_AUTH_USER_PASSWORD")
+
+global PASSWORD
+
+if AUTH_PROVIDER == "activeDirectory":
+    PASSWORD = AUTH_USER_PASSWORD
+else:
```

Comment:
Let's change it to `elif` as there will be a `freeIpa` at some point.

Reply:
We have to introduce the variables for free IPA password if we want to do that. Hence didn't do that.

Reply:
We don't have as `else` is optional but this is a minor thing that can be improved later on.

Reply:
Done